### ISSUE TITLE
No content of default metric in web UI 

### ISSUE THREAD

**Issue by zhengqinwen95** [microsoft/nni#3717]
**Describe the issue**:
 I run my code in a server container,  as shown below. 

`nvidia-docker run   -w /home/work \
                    -v xxx/nni:/home/work/nni \
                    -p 8125:8125 \
                    -p 51188:51188 \
                    -it xxx/nni_slim:test-0.0.6 /bin/bash`

`nnictl create -c /home/work/nni/examples/trials/mnist-pytorch/config.yml -p 8125`

Find no content in default metric :

![image](https://user-images.githubusercontent.com/82742349/120452728-fe286a80-c3c4-11eb-8c39-423d6a6ca928.png)

message of "nnictl trial ls" is normal
![image](https://user-images.githubusercontent.com/82742349/120452974-3334bd00-c3c5-11eb-8819-5782feccb6d9.png)

message in ~/nni-experiments/iaKdOZFM/trials/weOTg/.nni/metrics is normal
![image](https://user-images.githubusercontent.com/82742349/120453470-a6d6ca00-c3c5-11eb-8e5f-262d324eab97.png)

**Environment**:
- NNI version:
- Training service (local|remote|pai|aml|etc):
- Client OS: Mac OS
- Server OS (for remote mode only): Ubuntu16.04
- Python version:
- PyTorch/TensorFlow version:
- Is conda/virtualenv/venv used?:
- Is running in Docker?:

**Configuration**:
 - Experiment config (remember to remove secrets!):
 - Search space:

**Log message**:
 - nnimanager.log: 
 - dispatcher.log:
 - nnictl stdout and stderr: normal
 

**How to reproduce it?**:
Prepare two machines, one machine starts NNI in container and the other machine accesses the UI

**Comment by SparkSnail** [comment]
Could you start your experiment using `nnictl create --debug` , and upload nniManager.log here?

**Comment by zhengqinwen95** [comment]
[nnimanager.log](https://github.com/microsoft/nni/files/6595864/nnimanager.log)

**Comment by zhengqinwen95** [comment]
> Could you start your experiment using `nnictl create --debug` , and upload nniManager.log here?

[nnimanager.log](https://github.com/microsoft/nni/files/6595868/nnimanager.log)

**Comment by SparkSnail** [comment]
There is no [`Sending metrics`](https://github.com/microsoft/nni/blob/master/ts/nni_manager/training_service/local/localTrainingService.ts#L482) log information in log file, there should be some unknown issue when nodejs listen file in this container, it may caused by OS issue.
For workaround here, you could use `remote` mode to start experiment, set remoteIp as 127.0.0.1 to connect machine. In remote mode, NNI will not use file stream to record metric and send metrics by restful server.

**Comment by scarlett2018** [comment]
> There is no [`Sending metrics`](https://github.com/microsoft/nni/blob/master/ts/nni_manager/training_service/local/localTrainingService.ts#L482) log information in log file, there should be some unknown issue when nodejs listen file in this container, it may caused by OS issue.
> For workaround here, you could use `remote` mode to start experiment, set remoteIp as 127.0.0.1 to connect machine. In remote mode, NNI will not use file stream to record metric and send metrics by restful server.

@zhengqinwen95 - had you tried this suggestion? does it work for you? may we close the issue? thanks.

**Comment by zhengqinwen95** [comment]
> > There is no [`Sending metrics`](https://github.com/microsoft/nni/blob/master/ts/nni_manager/training_service/local/localTrainingService.ts#L482) log information in log file, there should be some unknown issue when nodejs listen file in this container, it may caused by OS issue.
> > For workaround here, you could use `remote` mode to start experiment, set remoteIp as 127.0.0.1 to connect machine. In remote mode, NNI will not use file stream to record metric and send metrics by restful server.
> 
> @zhengqinwen95 - had you tried this suggestion? does it work for you? may we close the issue? thanks.

I will try this method, you can close this issue, thanks for your help.
